CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * [#2176](https://github.com/ruby-grape/grape/pull/2176): Fixes issue-2175 - options call failing if matching all routes - [@myxoh](https://github.com/myxoh).
+* [#2177](https://github.com/ruby-grape/grape/pull/2177): Fixes issue-2094 - Issue with the order of `as:` and `default:` options - [@Catsuko](https://github.com/Catsuko).
```

Comment:
Not sure how we inherited this pattern, but we don't usually have one :), change this to `Fix: actual specific problem between as: and default: validators`, and for the line above `Fix: OPTIONS fails if matching all routes`. Thanks.

Reply:
sounds good, I reworded it so hopefully it describes it better